reporter-opentsdb/src/test/java/io/ultrabrew/metrics/reporters/opentsdb/OpenTSDBHttpClientTest.java

```diff
@@ -86,6 +87,23 @@ public void testWriteFailsManualFlush() throws Exception {
     assertEquals(0, client.currentBatchSize);
   }
 
+  @Test
+  public void testNullStringsAndTagsValidation() throws Exception {
+    new Expectations() {{
+      
+    }};
+    OpenTSDBHttpClient client = new OpenTSDBHttpClient(DUMMY_DB_URI, 64, true);
+    client.write(null, new String[] { "host", "web01" }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { null, "web01" }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host", null }, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host", "web01" }, 1534055562000000003L, null);
+    client.write("cpu_load_short.temp", null, 1534055562000000003L, "80");
+    client.write("cpu_load_short.temp", new String[] { "host" }, 1534055562000000003L, "80");
+    new Verifications() {{
+      times = 0;
```

Comment:
Is there something missing here? What are we verifying was called zero times?

Reply:
yup missing, doh.